app/views/transfer/new.html.erb

```diff
@@ -1 +1,100 @@
 <%= content_for(:title, "Thesis Transfer Submission | MIT Libraries") %>
+
+<% content_for :additional_js do %>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-validate/1.17.0/jquery.validate.min.js"></script>
```

Comment:
🙈 (and yes, I know this is how we do this already in this app so it is totally appropriate but also 🙈 )

Reply:
Heh - I've been faced with this exact problem in some personal projects, and noted that this is how it happens here. From a design perspective, it would be interesting to discuss a better way of doing this at some point.

But yes, a bit of 🙈 for now.